server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -365,18 +365,18 @@ private void recover() throws StorageGroupProcessorException {
   private void recoverCompaction() {
     if (!CompactionMergeTaskPoolManager.getInstance().isTerminated()) {
       compactionMergeWorking = true;
-      logger.info("{} submit a compaction merge task", virtualstorageGroupName);
+      logger.info("{} submit a compaction merge task", virtualstorageGroupId);
       try {
         CompactionMergeTaskPoolManager.getInstance()
             .submitTask(
                 tsFileManagement.new CompactionRecoverTask(this::closeCompactionMergeCallBack));
       } catch (RejectedExecutionException e) {
         this.closeCompactionMergeCallBack();
-        logger.error("{} compaction submit task failed", virtualstorageGroupName);
+        logger.error("{} compaction submit task failed", virtualstorageGroupId);
       }
     } else {
       logger.error("{} compaction pool not started ,recover failed",
-          virtualstorageGroupName);
+          virtualstorageGroupId);
```

Comment:
Storage group name is still missing in these logs. And please also check other places.

Reply:
Sure, I have fixed all the logs~